### PR TITLE
make top left nav home to app; move CfB logo  closes #542

### LINKED ISSUE #542: make top left nav home to app; move CfB logo
Common design patterns, noticed when an fellow member clicked it to try to go to the app home

### PR DESCRIPTION
**Pull request by misspran**

### CODE REVIEW

File: src/components/MainMenu.js [hunk 1]
```diff
@@ -17,15 +17,7 @@ const MainMenu = function (props) {
         inverted
         secondary
         size='large'>
-        <Menu.Item>
-          <a
-            href="http://www.codeforboston.org"
-            target="_blank"
-            rel="noopener noreferrer"><Image
-              src={ logo }
```
Comment (ethanbb) [inline src/components/MainMenu.js]: You can get rid of the `logo` and `Image` import statements in this file (lines 5 and 10) now that they're not being used here. I believe that's what's currently causing the Travis build to fail.

Comment (misspran) [reply src/components/MainMenu.js]: Thanks! Forgot about those.

File: src/components/Footer.js [hunk 2]
```diff
@@ -20,7 +22,7 @@ const Footer = () => {
         inverted
         stackable>
         <Grid.Row>
-          <Grid.Column width={ 7 }>
+          <Grid.Column width={ 4 } floated='left' left>
```
Comment (knod) [inline src/components/Footer.js]: The attribute `left` is causing errors. I believe it can just be removed.

File: src/components/Footer.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import logo from '../images/logo.svg';
 
 import {
   Grid,
   Header,
   Icon,
   Segment,
+  Image
```
Comment (knod) [inline src/components/Footer.js]: Note for us for followup (no worries to @misspran, we can take care of this one if you want): This, along with a couple other small things, are causing lint warnings. Someone should run `npm run lint` when we get a chance.

**Comment by knod** [comment]
Good PR! And thanks for fixing the import statements. A small change, and then I think we're good to go!

File: src/components/Footer.js (hunk 2 above)
Comment (knod) [reply src/components/Footer.js]: (They show up in the browser console, not in the terminal)

Comment (misspran) [reply src/components/Footer.js]: The left thing is placed in there to keep the styling from Semantic UI React. I dunno at least that's why I did it that way without using custom CSS. Unless we prefer to use custom CSS I'll try to do that instead.

Comment (knod) [reply src/components/Footer.js]: Really? When I removed it, `floated='left'` seemed to be sufficient. Maybe I'm wrong, though. I've been pretty tired.

Comment (misspran) [reply src/components/Footer.js]: Oh hmm, maybe just the left at the end is needed then. Sounds good! I'll remove it.

**Comment by knod** [comment]
Thanks!

**Comment by knod** [comment]
I guess it's testing the build as if it were already merged, not just the PR build, because this PR build is passing on my local branch. The error looks the same as the one everything else is getting, so I'm just going to go ahead and merge this.

**Pull request merged**